Stop overriding the app css_compressor setting

byroot opened the pull request:
I fail to understand why this gem is doing so, but it's preventing users to configure the CSS compressing as they will.

@bolandrm could we remove it, or am I missing something?

cc @edward

bolandrm commented:
Hey, looks like a bug.  I believe the only line that needs to be removed is Line 56.

byroot commented:
> Hey, looks like a bug. I believe the only line that needs to be removed is Line 56.

No, because `nil` mean you don't want any compressor. For example we don't want compression in our test environment.

bolandrm commented:
After removing line 56, the API becomes the same as sass-rails.  sass-rails appears to force compression in the test environment.

byroot commented:
God you're right. Let's do like sass-rails then.

But ping @rafaelfranca: Do you think this is a same default: https://github.com/rails/sass-rails/blob/master/lib/sass/rails/railtie.rb#L69-L76 ?

bolandrm commented:
@byroot I can incorporate this PR:

https://github.com/rails/sass-rails/pull/338/files

so that we allow no compression in the test environment.

byroot commented:
Sounds like a good idea. Maybe we should wait to see what `sass-rails` does of this PR?

In the meantime  I updated this PR to copy `sass-rails` behaviour, and I fixed the test suite.
